src/module-elasticsuite-core/Test/Unit/Search/Adapter/Elasticsuite/Request/Query/Builder/MatchTest.php

```diff
@@ -13,8 +13,8 @@
  */
 namespace Smile\ElasticsuiteCore\Test\Unit\Search\Adapter\Elasticsuite\Request\Query\Builder;
 
-use Smile\ElasticsuiteCore\Search\Request\Query\Match as MatchQuery;
-use Smile\ElasticsuiteCore\Search\Adapter\Elasticsuite\Request\Query\Builder\Match as MatchQueryBuilder;
+use Smile\ElasticsuiteCore\Search\Request\Query\QueryMatch as MatchQuery;
```

Comment:
If you name it "MatchQuery" instead of "QueryMatch" this will allow to get rid of this aliasing "QueryMatch as MatchQuery".

Also, since the builder was named "MatchQueryBuilder", this would be more consistent.

Reply:
Done.